modules/auth/register-service.ts

```diff
@@ -0,0 +1,15 @@
+import supabase from '@/services/auth/auth-supabase.service';
+
+export async function loginUser(email, password) {
+  try {
+    const { user, error } = await supabase.auth.signIn({
+      email,
+      password,
+    });
```

Comment:
I avoid to throw errors on the service layer, return an object with the result or `undefined` 

If is an array return empty array in case of error. Normally the error layer shall be in charge of doing a toast with the error message, then return some value so the flows continues 

Otherwise every time we use the API we need to remember to handle all the errors / message users